libnavigation-core/src/main/java/com/mapbox/navigation/core/reroute/RerouteOptionsAdapterManager.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.navigation.core.reroute
+
+internal interface RerouteOptionsAdapterManager {
```

Comment:
nit: We could live without this interface and just cast to our implementation of the reroute controller, this interface won't be used anywhere else anyway. Works either way though.

Reply:
👍